config/initializers/devise.rb

```diff
@@ -27,3 +27,34 @@
 
   config.timeout_in = 7.days
 end
+
+Warden::Manager.after_set_user do |record, warden, options|
+  next unless FeatureFlag.active?(:incident_eviction)
+
+  scope = options[:scope]
+  lra = warden.session(scope)['last_request_at']
+
+  # User is not affected
+  next unless record.id.in?(1..46) && record.instance_of?(Candidate)
```

Comment:
Minor question but is there a reason this is done after assigning `scope` and `lra`? Is it possible to do it just after the feature flag check?

Reply:
I don't think there is a reason for it, happy to change it.

Reply:
Resolved with https://github.com/DFE-Digital/apply-for-teacher-training/pull/9185/commits/fe92c1470ce9761fb5cbbeb32c556ac36a8c77ea